Doc/library/io.rst

```diff
@@ -728,6 +728,12 @@ than raw I/O does.
 
       Return :class:`bytes` containing the entire contents of the buffer.
 
+   .. method:: peek(size=0, /)
+
+      .. versionadded:: 3.13
+
+      Return bytes from the current position onwards but without advancing the
+      position. The number of bytes returned may be less or more than requested.
```

Comment:
Wait. Why can it *more* than requested?

Apparently size=0 means "read all". Please document it.

Reply:
This was just copied from the documentation for `BufferedReader.peek()`, but I agree that it could be written better. Changed now.

> Wait. Why can it _more_ than requested?

`BufferedReader.peek()` ignores the size argument and just returns whatever it has in its internal buffer.

Reply:
I would prefer that BytesIO.peek() documents its behavior, rather than BufferedReader.peek() behavior.

Reply:
Absolutely, it’s already changed.

(BTW, is there a policy on who clicks the "Resolve conversation" button?)